index.html

```diff
@@ -28,12 +28,12 @@
     <p>Hello World!</p>
 
     <section id="welcome-section">
-        <h1>Welcome to our portforlio site ;)</h1>
+        <h1>Welcome to our portforlio site </h1>
     </section>
 
     <section id='projects'>
         <div class="project-tile"></div>
-        <a href="#"></a>
+        <a href="#">Project Title</a>
     </section>
 
     <section id="contact">
```

Comment:
The list element in this section needs to be wrapped in its parent element. For example <ul> or <ol>.